Formula/profanity.rb

```diff
@@ -29,13 +31,18 @@ class Profanity < Formula
   depends_on "libotr" => :recommended
   depends_on "gpgme" => :recommended
   depends_on "terminal-notifier" => :optional
+  depends_on :python if build.with? "python-plugin-support"
```

Comment:
If it works with system Python we shouldn't even need this dependency; `:python` is used to require 2.7 or higher.  Right @ilovezfs ?